src/protocolgame.cpp

```diff
@@ -2390,6 +2404,9 @@ void ProtocolGame::sendAddCreature(const Creature* creature, const Position& pos
 	msg.addByte(0x00); // can change pvp framing option
 	msg.addByte(0x00); // expert mode button enabled
 
+	msg.addString("http://static.tibia.com/images/store/");
+	msg.addByte(g_config.getNumber(ConfigManager::MAX_PACKETS_PER_SECOND));
```

Comment:
What is you suggestion?

2016-02-13 14:24 GMT-02:00 Sam <notifications@github.com>:

> In src/protocolgame.cpp
> <https://github.com/otland/forgottenserver/pull/1715#discussion_r52829946>
> :
>
> > @@ -2390,6 +2404,9 @@ void ProtocolGame::sendAddCreature(const Creature* creature, const Position& pos
> >  	msg.addByte(0x00); // can change pvp framing option
> >  	msg.addByte(0x00); // expert mode button enabled
> >
> > +	msg.addString("http://static.tibia.com/images/store/");
> > +	msg.addByte(g_config.getNumber(ConfigManager::MAX_PACKETS_PER_SECOND));
>
> Using the value of MAX_PACKETS_PER_SECOND is wrong though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/otland/forgottenserver/pull/1715/files#r52829946>.
>